docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -425,31 +478,46 @@ QGroundControl follow the steps below.
 
```

Comment:
the image says "Quadrotor x" but the doc says "Quadrotor X"
not a huge deal, but might be worth changing for consistency.